src/components/checkbox/checkbox.deps.js

```diff
@@ -0,0 +1,5 @@
+({
+    shouldDeps: [
+        { mods: { type: 'button' } }
```

Comment:
@nik-kor this deps on our level doesn't helps resolve template for `checkbox_type_button`, but it should work ;-)